pdns/dnsdistdist/docs/rules-actions.rst

```diff
@@ -1887,6 +1896,12 @@ The following actions exist.
   Before 1.7.0 this action was performed even when the query had been received over TCP, which required the use of :func:`TCPRule` to
   prevent the TC bit from being set over TCP transports.
 
+.. function:: TCResponseAction()
+
+  .. versionadded:: 1.9.0
+
+  Truncate an existing answer, to force the client to TCP.
```

Comment:
```suggestion
  Truncate an existing answer, to force the client to TCP.
  Only applied to UDP answers.
```

Reply:
Does it actually truncate or does it set the TC bit _without_ modifying the data pat of the packet?

Reply:
It does actually truncate.

Reply:
Alright, I'll add it to the documentation.